algorithm/cgal/cgal_browser.cjs

```diff
@@ -1,12 +1,12 @@
 
-var Module = (function() {
+var Module = (() => {
```

Comment:
Why the change of style? Just more compact?

Reply:
I think this was automatically generated by emscripten.